packages/visual-editor/src/components/editor/SpacingSelector.tsx

```diff
@@ -79,6 +79,7 @@ const convertDefaultSpacingsToOptions = (
 
 export const SpacingSelector = (
   spacingType: "padding" | "gap",
+  includeDefault: boolean,
   label?: string
```

Comment:
I feel like we should make this required instead of defaulting to the spacingType. Then I'd make it the first prop to be consistent with the BasicSelector.